eng/common/testproxy/test-proxy-tool.yml

```diff
@@ -6,14 +6,11 @@ steps:
         $(Build.SourcesDirectory)/eng/common/scripts/trust-proxy-certificate.ps1
     displayName: 'Language Specific Certificate Trust'
 
-  - pwsh: |
-      Write-Host "##vso[task.setvariable variable=OriginalPath]$env:PATH"
-    displayName: 'Store Path Value'
-
   - pwsh: |
       Write-Host "##vso[task.setvariable variable=ASPNETCORE_Kestrel__Certificates__Default__Path]$(Build.SourcesDirectory)/eng/common/testproxy/dotnet-devcert.pfx"
       Write-Host "##vso[task.setvariable variable=ASPNETCORE_Kestrel__Certificates__Default__Password]password"
-    displayName: 'Configure Kestrel Environment Variables'
+      Write-Host "##vso[task.setvariable variable=PROXY_MANUAL_START]true"
```

Comment:
Assumming previous PR will be in. Which is coming true as soon as I can push the common PRs through the pipe.